md/access-control-api.md

```diff
@@ -0,0 +1,61 @@
+# Access control API
+
+::: info
+**Note:** This API is only available in Efficiency and Performance edition.
+:::
+
+## BDM
+
+#### Description
+
+Access control is available to protect the tenant BDM. You can use this API to get the access control status (lastUpdatedBy, lastUpdateDate...).
+
+#### Methods
+ * GET - Read access control status.
+ * DELETE - Remove access control
+ 
+::: info
+**Note:** A servlet to import access control file exists. You can use it with this url `services/bdmAccessControl/install?bdmAccessControlUpload=accessControlFileToUpload.xml`. 
+  The upload needs to be performed before (POST portal/bdmAccessControlUpload).
```

Comment:
the file upload needs...
I would add "Use the file name returned in the response of the upload request to perform the import request (bdmAccessControlUpload parameter)"